app/api/helpers/mail.py

```diff
@@ -594,9 +597,11 @@ def send_order_purchase_organizer_email(order, recipients):
         site_link=order.event.site_link,
         order_url=order.site_view_link,
         event_date=order.event.starts_at.strftime('%d %B %Y'),
-        event_time=order.event.starts_at.strftime("%H:%M %Z"),
-        purchase_time=order.completed_at,
+        event_time=order.event.starts_at.replace(tzinfo=pytz.timezone('UTC'))
+        .astimezone(pytz.timezone(order.event.timezone))
```

Comment:
this will change the time to event timezone or local timezone?

Reply:
to event timezone. what do you mean by local timezone??

Reply:
I mean, the problem statement is to make it timezone aware.

Reply:
yes, change it to the timezone of the event.

Reply:
@mariobehling can clear your doubt here

Reply:
I don't have any doubt here.

Reply:
Time should be in event timezone.

Reply:
In this case, changes in filters.py are not required, they will affect other areas. So please only make changes in helpers/mail.py @progmatic-99

Reply:
In other areas too, event timezone is not shown. If I change back the filter, we would encounter the bug again.